var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -102,36 +123,24 @@ def install(self, spec, prefix):
         # NOTE: Early versions of Zoltan come packaged with a few embedded
         # library packages (e.g. ParMETIS, Scotch), which messes with Spack's
         # ability to descend directly into the package's source directory.
-        source_directory = self.stage.source_path
-        if spec.satisfies('@:3.6'):
-            zoltan_directory = 'Zoltan_v{0}'.format(self.version)
-            source_directory = join_path(source_directory, zoltan_directory)
-
-        build_directory = join_path(source_directory, 'build')
-        with working_dir(build_directory, create=True):
-            config = Executable(join_path(source_directory, 'configure'))
-            config(
-                '--prefix={0}'.format(prefix),
-                '--with-cflags={0}'.format(' '.join(config_cflags)),
-                '--with-cxxflags={0}'.format(' '.join(config_cflags)),
-                '--with-fcflags={0}'.format(' '.join(config_cflags)),
-                *config_args
-            )
-
-            # NOTE: Earlier versions of Zoltan cannot be built in parallel
-            # because they contain nested Makefile dependency bugs.
-            make(parallel=not spec.satisfies('@:3.6+fortran'))
-            make('install')
-
-        # NOTE: Unfortunately, Zoltan doesn't provide any configuration
-        # options for the extension of the output library files, so this
-        # script must change these extensions as a post-processing step.
-        if '+shared' in spec:
-            for lib_path in glob.glob(join_path(prefix, 'lib', '*.a')):
-                lib_static_name = os.path.basename(lib_path)
+        config_args.append('--with-cflags={0}'
+                           .format(' '.join(config_cflags)))
+        config_args.append('--with-cxxflags={0}'.
+                           format(' '.join(config_cflags)))
+        config_args.append('--with-fcflags={0}'.
+                           format(' '.join(config_cflags)))
```

Comment:
The formatting here is much worse than came before.  Can you please just do:
```
config_args.extend([
    '--prefix={0}'.format(prefix),
    ...
])
```

Reply:
Note that you don't have to specify `--prefix` now that this is an `Autotools` package. If you could be consistent with the style that would be good though - for example:

```
        config_args.append('--with-cflags={0}'
                           .format(' '.join(config_cflags)))
        config_args.append('--with-cxxflags={0}'
                           .format(' '.join(config_cflags)))
        config_args.append('--with-fcflags={0}'
                           .format(' '.join(config_cflags)))
```

Reply:
> 
> 
> The formatting here is much worse than came before. Can you please just do:
> 
> ```
> config_args.extend([
>     '--prefix={0}'.format(prefix),
>     ...
> ])
> ```

I'm rewite it.